docs/source/using/profiles.adoc

```diff
@@ -8,29 +8,47 @@ include::variables.adoc[]
 
 toc::[]
 
-[[pofile-overview]]
+[[profile-overview]]
 == Overview
 
-{project} allows to create multiple instances of {project}.
-Each {project} instance can be created with different configurations and users can switch between different instances.
+The profile feature allows creating multiple instances of {project}. 
 
-[[creating-profile]]
+A profile is an instance of the {project} VM along with all of its configuration and state.
+
+Each {project} profile can be created with its own configuration (memory, CPU, disk size, add-ons, and so on) and completely independent of other profiles.
+Profiles share nothing between them.
+Things like cache, add-ons and persistent config are specific to a single profile.
+This means if you have set some specific configuration parameters for a profile, these settings do not apply to other profiles.
```

Comment:
These four sentences feel like they're saying essentially the same thing in slightly different (more nuanced) ways. At some point, I'd like to rewrite this so that we convey the same information without feeling like we're repeating ourselves.

Reply:
I think now is the time. Let's do this right to start with.

Reply:
I think it could be as simple as:

"Each {project} profile is created with its own configuration (memory, CPU, disk size, add-ons, and so on) and is independent of other profiles. If you have common configuration options you want to apply to all profiles, you have to do this manually per profile. See xref:../using/profiles.adoc#global-config-profiles[Global Configuration Options]"

Effectively we skip the three repeating sentences.

Reply:
That works for me as long as the same essential information is being conveyed to the reader.